custom_components/xiaomi_cloud_map_extractor/common/image_handler.py

```diff
@@ -181,6 +185,57 @@ def draw_texts(image: ImageData, texts):
     def draw_layer(image: ImageData, layer_name):
         ImageHandler.__draw_layer__(image, image.additional_layers[layer_name])
 
+    @staticmethod
+    def __draw_image__(image: ImageData, center, r, outline, fill):
```

Comment:
This method is unnecessary

Reply:
Ah yes, I started developing it when I was trying to use an image but then dropped the idea and forgot to remove it.